### PR TITLE
Detailed publishing

### PR DESCRIPTION
**Pull request by Hootyoo**
Added step-by-step publishing guides for

- CloudFront
- CloudFlare
- nginx

### CODE REVIEW

**Review by zmeggyesi — CHANGES_REQUESTED** [review]
This really ought to be added to one of the TOC files as well, I had quite a hard time finding it initially.

File: dashboard2/publishwebsite/integration-guide/cloudfront.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+When you set up subdirectory publishing via CloudFront, you don't just set up a reverse proxy. CloudFront isn't just a simple web server solution after all. Instead of that, following this guide you'll set up a complete CDN in front of both your original and translated website. As such, you can also benefit from the other advantages that a CDN provides such as load balancing and global caching.
+
+To get started, it's recommended to copy the settings provided in the Publishing wizard. These are partially project-specific. We'll use a project with the following details:
+
+- Source domain: example.com
+- Project code: `redacted`
+- Translations exist for German
+- We wish to publish them to example.com/de/
+- Your white label is app.translationproxy.com
```
[the remark this answers is not in the document]
Comment (Hootyoo) [reply dashboard2/publishwebsite/integration-guide/cloudfront.md:11]: I was trying to come up with one that's sooo generic that I didn't expect anyone to have it. Would `app.proxytranslation.com` be suitable?

File: dashboard2/publishwebsite/integration-guide/cloudfront.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+8. On *Origins*, 
+    - Ensure that the `translationproxy-language` origin we created above is present
+    - Click *Create origin* and enter the following:
+        - Original domain: the source site, in this example it should be `example.com`.
+        - Protocol: HTTPS only is recommended, but Match viewer can work too.
+        - Origin path: leave empty.
+        - Name: enter a memorable name. We recommend the domain of the original site `example.com`.
+        - Click *Create origin* to save.
+    - **NOTE:** If you later wish to add additional languages, you don't need to create a completely different distribution, just add them here and follow the note under the following step too.
+9. On *Behaviors*, (which should really be spelled Behavio**u**rs, my spellchecker is complaining)
```
[the remark this answers is not in the document]
Comment (Hootyoo) [reply dashboard2/publishwebsite/integration-guide/cloudfront.md:63]: No fun allowed. Comment is removed.

File: dashboard2/publishwebsite/integration-guide/nginx.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+$ sudo su
+```
+
+**WARNING:** At this point, you have complete, *unrestricted* access to the server. It's very easy to break it. Don't run commands that you aren't certain will do what you need to.
+
+If you have no reason not to do so, it's probably a good idea to update your server.
+
+```
+# apt update && apt upgrade -y
+```
```
[the remark this answers is not in the document]
Comment (Hootyoo) [reply dashboard2/publishwebsite/integration-guide/nginx.md:54]: I didn't even mention `apt dist-upgrade` or `do-release-update`... In any case, I removed these lines.

File: dashboard2/publishwebsite/integration-guide/cloudfront.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zmeggyesi) [reply dashboard2/publishwebsite/integration-guide/cloudfront.md]: Yes, that would work.

File: dashboard2/publishwebsite/integration-guide/nginx.md [hunk 4]
```diff
@@ -159,7 +153,15 @@ The configuration is in place, but nginx doesn't automatically switch them over.
...
 ```
 
-If no errors are printed, you can reload without dropping connections via 
+If no errors are printed, you should verify that your server can access the translation proxy. You can do so, for example, via `curl` and `head`. Don't forget to replace `de-de-redacted.app.proxytranslation.com` with your temporary domain.
+
+```
+# curl -Is https://de-de-redacted.app.proxytranslation.com/ | head -1
+```
+
+This should print that the status code is `200`. If it doesn't, a likely explanation is that your server is missing Certificate Authority certificates and as a result can't connect to our service via HTTPS. Installing such certificates is beyond the scope of this article, but Tech Republic has a comprehensible tutorial on the topic [here](https://www.techrepublic.com/article/how-to-install-ca-certificates-in-ubuntu-server/).
```
Comment (zmeggyesi) [inline dashboard2/publishwebsite/integration-guide/nginx.md:162]: This is pretty good!

**Review by zmeggyesi — APPROVED** [review]

**Pull request merged**